### ISSUE TITLE
Qjackctl not displaying

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-ui#107]
- ssh to Zynthian
- qjackctl

qjackctl seems to be running but the windows do not render in my x-server. Same if it point at a different x-server, e.g.
DISPLAY=192.168.1.8:0 qjackctl.

On my Windows PC running VcXsrv (which works for other applications forwarded from the zynthian) I see two windows (default and connections) in the task bar but neither will render on the screen.

**Comment by jofemodo** [comment]
Are you using a ZynthianOS image? Please, confirm ;-)

**Comment by riban-bw** [comment]
I can't remember but probably Gorgona. Is there a way to tell?

**Comment by jofemodo** [comment]
Yes. The OS line in the webconf dashboard. If it's "Jessie", then it's a Gorgona. If it's  "Stretch", then is Aruk ;-)
